packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -123,6 +128,16 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
     scrollProperty: 'scrollY',
   });
 
+  const shouldShowLogin = !user && utmSource && isAuthReady;
+
+  useEffect(() => {
+    if (!shouldShowLogin) {
+      return;
+    }
+
+    showLogin({ trigger: AuthTriggers.FromNotification });
```

Comment:
```suggestion
    showLogin({ trigger: AuthTriggers.FromNotification, isLogin: true });
```
Think this should open the modal on the login view, and not the signup view

Reply:
Good catch. Yes, that is right.